src/api/controllers/reports-controller.ts

```
@@ -0,0 +1,145 @@
+      draft: false,
+      startDate,
+      endDate,
+    })
+
+    const { revenue, profit } = orders.reduce<{
+      revenue: number
+      profit: number
+    }>(
+      (acc, order) => {
+        return {
+          ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)